.github/ISSUE_TEMPLATE/feature_request.yml

```diff
@@ -0,0 +1,71 @@
+name: Feature request
+description: Request a new feature to be added or improved in Godot Tools VS Code add-on
+labels:
+  - enhancement
+body:
+
+- type: markdown
+  attributes:
+    value: |
+      - Write a descriptive issue title above.
+      - Search [open](https://github.com/godotengine/godot-vscode-plugin/issues) and [closed](https://github.com/godotengine/godot-vscode-plugin/issues?q=is%3Aissue+is%3Aclosed) issues to ensure it has not already been reported.
+      - Verify that you are using a [supported Godot version](https://docs.godotengine.org/en/stable/about/release_policy.html).
+      - This repository only contains the LSP *client*. Issues pertaining to the Godot editor itself or the LSP's *server* side should be reported on the [main Godot repository](https://github.com/godotengine/godot/issues).
```

Comment:
Not sure if ***issue*** is adequate to this context

Reply:
A feature request is still an issue, technically :slightly_smiling_face: